worlds/messenger/rules.py

```diff
@@ -430,6 +432,9 @@ def __init__(self, world: "MessengerWorld") -> None:
             {
                 "Autumn Hills Seal - Spike Ball Darts":
                     lambda state: self.has_vertical(state) and self.has_windmill(state) or self.is_aerobatic(state),
+                "Autumn Hills Seal - Double Swing Saws":
+                    lambda state: self.has_vertical(state) or self.has_windmill(state) or
```

Comment:
Does `self.has_windmill(state)` do anything here? Isn't that incorporated into `can_destroy_projectiles`?

Reply:
ah yeah good catch